pyanaconda/ui/gui/__init__.py

```diff
@@ -289,6 +290,7 @@ def __init__(self, fullscreen=False, decorated=False):
         # Remove the title bar, resize controls and other stuff if the window manager
         # allows it and decorated is set to False. Otherwise, it has no effect.
         self.set_decorated(decorated)
+        self.set_titlebar(Gtk.Invisible())
```

Comment:
```suggestion
        self.set_titlebar(None)
```

This should resolve the issue we are seeing in tests. 

Suggested by @halfline , thanks a lot!

Reply:
I tested it briefly locally and it looks it works fine. However, I need to test it properly on Live ISOs.

Reply:
No, that's exactly the same as not setting a titlebar, thus Gtk will ask the compositor to add one

Reply:
Apparently `set_decorated(False)` makes any titlebar widget invisible.
So you can set any other widget as titlebar and this will work fine.
I think `Gtk.DrawingArea()` might be a good candidate as a very minimal widget.